lib/evaluate/characteristics.h

```diff
@@ -198,9 +235,16 @@ struct Procedure {
   bool operator==(const Procedure &) const;
 
   // Characterizes the procedure represented by a symbol, which may be an
-  // "unrestricted specific intrinsic function".
+  // "unrestricted specific intrinsic function".  The Boolean argument
+  // (when true) is used to treat an external function or subroutine
+  // in the same source file as if it did not have a known explicit interface
+  // when it is being called from a distinct program unit.
```

Comment:
What boolean argument?

Reply:
The one that used to be there but no longer is.  I'll fix the comment.